test/e2e/toolbox_test.go

```diff
@@ -0,0 +1,298 @@
+package integration
+
+/*
+	toolbox_test.go is under the care of the Toolbox Team.
+
+	The tests are trying to stress parts of Podman that Toolbox needs for
+	its functionality.
+
+	https://github.com/containers/toolbox
+
+	Toolbox Team:
+	Ondřej Míchal <harrymichal@fedoraproject.org>
+	Debarshi Ray <rishi@fedoraproject.org>
+
+	Also available on Freenode IRC on #silverblue or #podman,
+*/
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+	"time"
+
+	. "github.com/containers/podman/v2/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Toolbox-specific testing", func() {
+	var (
+		tempdir    string
+		err        error
+		podmanTest *PodmanTestIntegration
+	)
+
+	BeforeEach(func() {
+		tempdir, err = CreateTempDirInTempDir()
+		if err != nil {
+			os.Exit(1)
+		}
+		podmanTest = PodmanTestCreate(tempdir)
+		podmanTest.Setup()
+		podmanTest.SeedImages()
+	})
+
+	AfterEach(func() {
+		podmanTest.Cleanup()
+		f := CurrentGinkgoTestDescription()
+		processTestResult(f)
+	})
+
+	It("podman run --dns=none - allows self-management of /etc/resolv.conf", func() {
+		var session *PodmanSessionIntegration
+
+		session = podmanTest.Podman([]string{"run", "--dns", "none", ALPINE,
+			"rm", "/etc/resolv.conf", ";",
+			"touch", "/etc/resolv.conf"})
```

Comment:
Does this syntax actually work? The usual convention seems to be:
```go
    podmanTest...( "sh", "-c", "this;that;theother")
```
...but this was just a detour for bringing me to my favorite rant: "run something and only check exit code" is not a test. Some typos can lead to a command passing but not actually doing anything. A much better idea is to check for unusual strings. For instance:
```
sh -c "rm /etc/resolv.conf && touch -d '1970-01-01 00:02:03' /etc/resolv.conf && stat -c %s:%Y /etc/resolv.conv"
```
...and confirm that you get an exact match on `0:123`

Reply:
> Does this syntax actually work? The usual convention seems to be:

Yes, I was not sure entirely how to go about the commands in the entry-point. Your idea seems to be the right one.

> ...but this was just a detour for bringing me to my favorite rant: "run something and only check exit code" is not a test. 

Roger! The next iteration should fix this. Thank you!